x-pack/plugins/security_solution/public/resolver/view/clickthrough.test.tsx

```diff
@@ -147,7 +151,10 @@ describe('Resolver, when analyzing a tree that has two related events for the or
     it('should render a related events button', async () => {
       await expect(
         simulator.map(() => ({
-          relatedEventButtons: simulator.processNodeRelatedEventButton(entityIDs.origin).length,
+          relatedEventButtons: simulator.processNodeChildElements(
```

Comment:
Whats the purpose of the special treatment here?

Reply:
Same as below. Retain the original intention here. I could move all of the actual `selector` string into this test or maybe even export the `processNodeElementSelector` and build the string at this level. Alternatively, I could tie the actual selector with the associated `entity-id` (see below comment)